### DISCUSSION TITLE
Welcome to neutralinojs Discussions!

### DISCUSSION THREAD

**Discussion by shalithasuranga** [neutralinojs/neutralinojs#336] [General]
👋 Welcome!

If you have anything to discuss, here is the place...

**Comment by xbais** [comment]
Hello there! Can someone please share with me an in-depth documentation for NeutralinoJS which can be downloaded as a PDF file for personal reference.
Thanks

**Reply by MyBlood** [reply]
> Hello there! Can someone please share with me an in-depth documentation for NeutralinoJS which can be downloaded as a PDF file for personal reference.
> Thanks

why don't you just read it online here?
https://neutralino.js.org/docs/
